wpilib/wpilib/sendablebuilder.py

```diff
@@ -146,7 +146,6 @@ def addBooleanProperty(self, key, getter, setter):
         :param getter:  getter function (returns current value)
         :type getter: () -> bool
         :param setter:  setter function (sets new value)
-        :type setter: (bool) -> Any
```

Comment:
y?

Reply:
Cause it doesn't necessarily take a bool...although I thought I replaced it with...something.